magma_access_gateway_installer/agw_preinstall_checks.py

```diff
@@ -45,11 +45,11 @@ def preinstall_checks(self):
     def install_required_system_packages(self):
```

Comment:
should we install stuff in the preinstall_checks section?

Reply:
I've changed the name of the class to `AGWInstallerPreinstall` and the module's name to agw_preinstall. 
Alternative could be to write a method checking if there are missing/outdated required packages and installing them if needed. The downside of this option is that we'd have to check for both missing and outdated packages. This would complicate the code, whereas simple install covers both cases. I don't think it's worth the effort.
Let me know what you think.

Reply:
Agreed